tron/core/actionrun.py

```diff
@@ -257,47 +257,64 @@ class ActionRun(Observable, Persistable):
     WAITING = "waiting"
     UNKNOWN = "unknown"
 
-    default_transitions = dict(fail=FAILED, success=SUCCEEDED)
+    default_transitions = {"fail": FAILED, "success": SUCCEEDED}
```

Comment:
i took the liberty of making these inline dicts to make it easier to update things (and 'cause it's 2025 - we shouldn't be using the dict() constructor anymore)